spec/support/activity_shared_examples.rb

```diff
@@ -54,32 +54,3 @@
     expect { submit_form }.not_to have_enqueued_job(ActivityTrackingJob)
   end
 end
-
-# Define the following let variables before using these examples:
-#
-#   create_activities : a block which creates the activities AND IS CALLED
-#                       BEFORE THE PAGE LOADS
-#   trackable: the model which the 'show' page is about
-shared_examples 'a page with an activity feed' do
```

Comment:
I'm realizing that we still have this feed but it's now in the revisions view. In that case we still need to have a spec for that.

Reply:
added specs